### PR TITLE
Feature - Add layers button and tooltips to map control buttons

### PR DESCRIPTION
**Pull request by inefoss**
image

### CODE REVIEW

File: src/components/Map/control/Controls.tsx [hunk 1]
```diff
@@ -48,9 +48,9 @@ function RegisterEchoSideDrawers() {
             {
                 panelType: PanelType.right,
                 component: () => (
-                    <LayerControlSideDrawer layers={context.layers} />
```
Comment (Tobzor) [inline src/components/Map/control/Controls.tsx:51]: Some formatting issues across IDÈs here i assume :(

**Review by Tobzor — APPROVED** [review]

**Review by ErlendAHall — APPROVED** [review]

File: .eslintrc.js [hunk 2]
```diff
@@ -21,5 +21,17 @@ module.exports = {
...
+        'prettier/prettier': [
+            'error',
+            {
+                //see: https://prettier.io/docs/en/options.html
+                semi: true,
+                trailingComma: 'all',
+                singleQuote: true,
+                printWidth: 80,
+                tabWidth: 4,
+                endOfLine: 'auto',
```
Comment (ErlendAHall) [inline .eslintrc.js:33]: eof's should default to `lf`? Otherwise, codebases will have a mix of `lf` and `crlf`

**Review by ErlendAHall — DISMISSED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (inefoss) [reply .eslintrc.js]: Updated it now

Comment (inefoss) [reply .eslintrc.js]: For eslint and prettier

**Review by Tobzor — APPROVED** [review]

**Pull request merged**